server/server.js

```diff
@@ -1,25 +1,110 @@
 const express = require('express');
+
 const path = require('path');
+
 const app = express();
-const port = 8080;
+const port = 8070;
+const mysql = require('mysql');
+// const alert = require('alert');
+
+const bodyParser = require('body-parser');
+
+// const { check } = require('express-validator');
+
+app.use(bodyParser.urlencoded({ extended: false }));
```

Comment:
Should probably add a comment to explain the purpose of this line (I'm unsure what it's doing)